python/tskit/trees.py

```diff
@@ -5445,6 +5455,17 @@ def draw_svg(
 
         :return: An SVG representation of a tree sequence.
         :rtype: str
+
```

Comment:
Do we need this note? I can't imagine this tripping anyone up and the docs for svg drawing are already a lot?

Reply:
Yeah, when the viz tutorial is done you can do a reduction/interlinking pass through these docs.